_includes/layouts/company.njk

```diff
@@ -0,0 +1,5 @@
+---
+layout: layouts/base.njk
+---
+
+<h1>{{name}}</h1>
```

Comment:
In the future the company page will have more data but I just wanted to show the name to prove that it works for now.